src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -13292,6 +13292,96 @@ .maxstack  4
 ");
         }
 
+        [WorkItem(24348, "https://github.com/dotnet/roslyn/issues/24348")]
+        [Fact]
+        public void VarargBridgeSource()
```

Comment:
Consider merging the tests.
The first test is easily implemented inside the second one (`CompileAndVerify(lib_cs + client_cs, ...`).

Reply:
Before the fix one test would pass, and another would fail. It also easy to get into a regression where either both start failing or only one. I had such cases due to mistakes while fixing the bug.
I figured, it is better to keep separate tests so that it would be easy to diagnose what went wrong.

---
In reply to: [166780870](https://github.com/dotnet/roslyn/pull/24679#discussion_r166780870) [](ancestors = 166780870)